cli/src/semgrep/formatter/sarif.py

```diff
@@ -275,16 +279,32 @@ def _rule_to_sarif(rule: Rule) -> Mapping[str, Any]:
             rule_json = {
                 "id": rule.id,
                 "name": rule.id,
-                "shortDescription": {"text": rule.message},
+                "shortDescription": {"text": f"Semgrep Finding: {rule.id}"},
                 "fullDescription": {"text": rule.message},
                 "defaultConfiguration": {"level": severity},
                 "properties": {"precision": "very-high", "tags": tags},
             }
 
         rule_url = rule.metadata.get("source")
+        references = []
+
         if rule_url is not None:
             rule_json["helpUri"] = rule_url
-
+            references.append(f"[Semgrep Rule]({rule_url})")
+
+        if rule.metadata.get("references"):
+            ref = rule.metadata["references"]
+            references.extend(
+                [f"[{r}]({r})" for r in ref]
+                if isinstance(ref, list)
+                else [f"[{ref}]({ref})"]
+            )
```

Comment:
nit: are we 100% sure these references will always be URLs? if so, I think we can just add them without the markdown links

Reply:
In semgrep rules they will always be urls

Reply:
Oh gotcha 👍